README.md

```diff
@@ -47,9 +47,23 @@ For the quickstart and Docker installation modes, [Docker](https://docker.com) n
 Please refer to the official documentation and make sure your user is added to the `docker` user group if you cannot
 access root permissions to run containers.
 
-### Backups
+### Security
+
+For validators it is crucial to safely back up their private key. If they lose their key, they will not be able to
+
+- create any blocks or earn block rewards or
```

Comment:
Thought about it for a bit, but I think it's logically correct: the sentence has the structure `not (A or B)`, so this expression can be used to both infer both `not A` and `not B`. If we'd write `not (A and B)`, it could be that one of `A` and `B` is true.

Reply:
Haha, nice elaborated. :joy: